src/components/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  CardWrapper,
+  ProductImageWrapper,
+  ProductImage,
+  ProductName,
+  ProductCategory,
+  ProductPrice,
+  AddToCartButton,
+  AddToCartButtonWrapper
+} from './ProductCard.styles';
+
+export const ProductCard = ({image, name, category, price}) => {
+  name = name.length > 45 ? name.slice(0, 45) + '...' : name; // To keep the product name max in 2 rows when rendering in ProductCard.
+  price = '$' + price;
+
+  return (
+  <CardWrapper>
+
+    <ProductImageWrapper>
+      <ProductImage src={image} alt={'Picture of a ' + name} />
+    </ProductImageWrapper>
+
+    <ProductName>{name}</ProductName>
+    <ProductCategory>{category}</ProductCategory>
+    <ProductPrice>{price}</ProductPrice>
```

Comment:
You can also add the`$` symbol instead of modifying the value that comes in `price = '$' + price;`

`<ProductPrice>$ {price}</ProductPrice>`